integrations/opsgenie/utils.py

```diff
@@ -0,0 +1,15 @@
+from enum import StrEnum
+
+
+class ObjectKind(StrEnum):
+    ALERT = "alerts"
+    SERVICE = "services"
+    TEAM = "teams"
+
+
+# A dictionary to map each resource type to its API version
+RESOURCE_API_VERSIONS = {
+    ObjectKind.ALERT: "v2",
+    ObjectKind.SERVICE: "v1",
+    ObjectKind.TEAM: "v2",
```

Comment:
why dont you export team sync?
why are we exporting only alerts and services? there are alot more kinds in ops genie API

Reply:
I'll sync the team then